content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_valuation/integrating_landed_costs.rst

```diff
@@ -1,81 +1,168 @@
-=======================================================
+============
+Landed costs
+============
 
 .. _inventory/reporting/landed_costs:
 
-The landed cost feature in Odoo allows the user to include additional costs (shipment, insurance,
-customs duties, etc.) into the cost of the product.
+.. |RfQ| replace:: :abbr:`RfQ (Request for Quotation)`
+.. |PO| replace:: :abbr:`PO (Purchase Order)`
+.. |FIFO| replace:: :abbr:`FIFO (First In First Out)`
+.. |AVCO| replace:: :abbr:`AVCO (Average Costing)`
+
+When shipping products to customers, the landed cost is the total price of a product or shipment,
+including all expenses associated with shipping the product.
+
+In Odoo, the *Landed Costs* feature is used to take additional costs into account when calculating
+the valuation of a product. This includes the cost of shipment, insurance, customs duties, taxes,
+and other fees.
 
 Configuration
 =============
 
-First, go to :menuselection:`Inventory --> Configuration --> Settings --> Valuation` and activate
-the :guilabel:`Landed Costs` feature. Odoo also gives the option to set a :guilabel:`Default
-Journal` in which the landed costs accounting entries will be recorded.
+To add landed costs to products, the *Landed Costs* feature must first be enabled. To enable this
+feature, navigate to :menuselection:`Inventory app --> Configuration --> Settings`, and scroll to
+the :guilabel:`Valuation` section.
+
+Tick the checkbox next to the :guilabel:`Landed Costs` option, and click :guilabel:`Save` to save
+changes.
+
+Once the page refreshes, a new :guilabel:`Default Journal` field appears below the :guilabel:`Landed
+Costs` feature in the :guilabel:`Valuation` section.
 
-.. image:: integrating_landed_costs/landed-costs-setting.png
+Click the :guilabel:`Default Journal` drop-down menu to reveal a list of accounting journals. Select
+a journal for which all accounting entries related to landed costs should be recorded.
+
+.. image:: integrating_landed_costs/integrating-landed-costs-enabled-setting.png
    :align: center
-   :alt: Activate the landed cost feature in Inventory settings.
+   :alt: Landed Costs feature and resulting Default Journal field in the Inventory settings.
 
-Add costs to products
-=====================
+Create landed cost product
+==========================
+
+For charges that are consistently added as landed costs, a landed cost product can be created in
+Odoo. This way, a landed cost product can be quickly added to a vendor bill as an invoice line,
+instead of having to be manually entered every time a new vendor bill is created.
+
+To do this, create a new product by going to :menuselection:`Inventory app --> Products -->
+Products`, and clicking :guilabel:`New`.
 
-Receive the vendor bill
------------------------
+Assign a name to the landed cost product in the :guilabel:`Product Name` field (i.e.
+International Shipping). In the :guilabel:`Product Type` field, click the drop-down menu, and select
```

Comment:
```suggestion
`International Shipping`). In the :guilabel:`Product Type` field, click the drop-down menu, and select
```
using single ticks here because I'm under the impression that's the formatting we use when suggesting that something be "entered" as a name/entry in a form field.

Reply:
Suggestion implemented ✅